apps/ledger-live-desktop/tests/specs/services/wallet-api.spec.ts

```diff
@@ -0,0 +1,127 @@
+import test from "../../fixtures/common";
+import { expect } from "@playwright/test";
+import { DiscoverPage } from "../../models/DiscoverPage";
+import { Layout } from "../../models/Layout";
+import { Drawer } from "../../models/Drawer";
+import { Modal } from "../../models/Modal";
+import { DeviceAction } from "../../models/DeviceAction";
+import * as server from "../../utils/serve-dummy-app";
+import { randomUUID } from "crypto";
+
+test.use({ userdata: "1AccountBTC1AccountETH" });
+
+let continueTest = false;
+
+test.beforeAll(async ({ request }) => {
+  // Check that dummy app in tests/utils/dummy-app-build has been started successfully
+  try {
+    const port = await server.start("dummy-wallet-app/build");
+    const url = `http://localhost:${port}`;
+    const response = await request.get(url);
+    if (response.ok()) {
+      continueTest = true;
+      console.info(
+        `========> Dummy Wallet API app successfully running on port ${port}! <=========`,
+      );
+      process.env.MOCK_REMOTE_LIVE_MANIFEST = JSON.stringify(
+        server.liveAppManifest({
+          id: "dummy-live-app",
+          url,
+          name: "Dummy Wallet API Live App",
+          apiVersion: "2.0.0",
+          content: {
+            shortDescription: {
+              en: "App to test the Wallet API",
+            },
+            description: {
+              en: "App to test the Wallet API with Playwright",
+            },
+          },
+        }),
+      );
+    } else {
+      throw new Error("Ping response != 200, got: " + response.status);
+    }
+  } catch (error) {
+    console.warn(`========> Dummy test app not running! <=========`);
+    console.error(error);
+  }
+});
+
+test.afterAll(() => {
+  delete process.env.MOCK_REMOTE_LIVE_MANIFEST;
```

Comment:
server.start in `beforeAll` but no server.end in `afterAll` ?

Are we sure we are not leaving a process open ?

Reply:
Good spot